eng/build.ps1

```diff
@@ -65,7 +65,7 @@ function Get-Help() {
   Write-Host "Libraries settings:"
   Write-Host "  -allconfigurations      Build packages for all build configurations."
   Write-Host "  -coverage               Collect code coverage when testing."
-  Write-Host "  -framework (-f)         Build framework: net5.0 or net472."
+  Write-Host "  -framework (-f)         Build framework: net5.0, net48 or any other TFM."
```

Comment:
Would we really support some other TFM being passed?  I can't imagine we do the right thing when someone passes netstandard2.1 here for instance.

Reply:
If you pass in netstandard2.1 you would build the best configuration if there's one for the projects. Meaning you would build netstandard2.1 and netstandard2.0 refs and sources. That already works today, or isn't that what you would expect to happen?

Reply:
That's what it does, but is it meaningful?  Would someone ever want to do this?  The parts that I was imagining wouldn't do the right thing are shims, pretest, any pkgs.

Reply:
shims and pretest are conditioned to only run on `$(NetCoreAppCurrent)`. Unsure about packages. For the purprose of this PR I will remove the "any other TFM" note.